src/screens/authentication/create_new_wallet/GetPublicKeyScreen/index.tsx

```diff
@@ -63,19 +65,50 @@ const GetPublicKeyScreen = ({ transport, setTransport }: Props) => {
       if (ledgerPublicKey) {
         setError(null);
         const keys = await getListKeys(casperApp, listKeys.length, 5);
-        setListKeys(keys);
+        dispatch(
+          allActions.user.getAccounts(
+            { publicKeys: keys },
+            async (err: any, data: any) => {
+              if (err) {
+                setError(err);
```

Comment:
The `err` variables is an object. The message will show like this.
![IMG_3616](https://user-images.githubusercontent.com/34257375/157460110-7c7ec009-a90f-4f53-9c38-98bbfcde9a9c.PNG)

Reply:
Fixed in https://github.com/CasperDash/casperdash-mobile-wallet/pull/52/commits/aeb97d73318455cc27fdbfe33b06966f58dfdad5